docs/installation/windows.md

```diff
@@ -159,6 +141,13 @@ NOTE: Make sure to download **the OSS version**, `elasticsearch-oss`.
 1. Fork DEV's repository, eg. <https://github.com/thepracticaldev/dev.to/fork>
 1. Clone your forked repository, eg.
    `git clone https://github.com/<your-username>/dev.to.git`
+1. Open the cloned dev.to folder in terminal with `cd dev.to`. Next, install
+   ruby with the following commands.
```

Comment:
Ruby should be capitalized, and I think we should use a colon instead of a period here:

`Next, install Ruby with the following commands:`

Reply:
Sounds good! I will add a note related to the role of `bin/setup` in the installation process. The rename of the `Ruby on WSL` title sounds appropriate too. Thank you for the pointers, Haha, these are the only docs I have worked on/contributed to, so still pretty new to the correct language to use.

I totally agree, the Windows docs definitely needs love since WSL is becoming easier to setup day by day. Anything for smoothing out the process for new contributors, still remember the week I spent understanding stuff when I began.